PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -294,25 +294,25 @@ local function CreateCamera()
 
 	function this:Update()
```

Comment:
If a camera type doesn't define it itself, it will use this. This ensures that it can be called without checking if it's nil. It's sort of an inheritance mechanism.